docs/content/development/datasketches-aggregators.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: doc_page
+---
+
+## DataSketches aggregator
+Druid aggregators based on [datasketches]()http://datasketches.github.io/) library. You would ingest one or more metric columns using either `sketchBuild` or `sketchMerge` aggregators. Then, at query time, you can use `sketchMerge` with appropriate post aggregators described below. Note that sketch algorithms are approxiate, see details in the [datasketches doc](http://datasketches.github.io/docs/theChallenge.html).
```

Comment:
names here have some historical significance as they were used since the inception of this module with many ppl using those.
That said, I think, it will be possible to have new names (with support for old names at the same time so that most of our client code does not break).
but I believe, names should have  `build` or `merge` in them so that it is clear whether they build a fresh sketch or just merge sketches (e.g. `sketchMerge` at ingestion time is used when user has already produced sketches as part of his/her batch pipeline, so input to druid already contains sketches) . Also having `ingest` in the name might be misleading some time e.g. `sketchMerge` aggregator is used both at ingestion time and query time.

yes, algorithm used is theta sketch a variant of KMV.